src/provider/provider_tracking.c

```diff
@@ -181,20 +190,21 @@ static umf_result_t trackingAllocationSplit(void *hProvider, void *ptr,
     tracker_value_t *value =
         (tracker_value_t *)critnib_get(provider->hTracker->map, (uintptr_t)ptr);
     if (!value) {
-        LOG_ERR("tracking split: no such value");
+        LOG_ERR("region for split is not found in the tracker");
         ret = UMF_RESULT_ERROR_INVALID_ARGUMENT;
         goto err;
     }
     if (value->size != totalSize) {
-        LOG_ERR("tracking split: %zu != %zu", value->size, totalSize);
+        LOG_ERR("tracked size %zu does not match requested size to split: %zu",
+                value->size, totalSize);
         ret = UMF_RESULT_ERROR_INVALID_ARGUMENT;
         goto err;
     }
 
     ret = umfMemoryProviderAllocationSplit(provider->hUpstream, ptr, totalSize,
                                            firstSize);
     if (ret != UMF_RESULT_SUCCESS) {
-        LOG_ERR("tracking split: umfMemoryProviderAllocationSplit failed");
+        LOG_ERR("umfMemoryProviderAllocationSplit failed");
```

Comment:
nit: In a few places we have log messages similar to `provider failed to split an allocation ...` and other times it's formulated like here `umfMemoryProviderAllocationSplit failed`.

Reply:
Done. I tried to use the first variant.